apps/server/src/modules/h5p-library-management/service/h5p-library-management.service.spec.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Test, TestingModule } from '@nestjs/testing';
+import { DeepMocked, createMock } from '@golevelup/ts-jest';
+import { ContentStorage, LibraryStorage } from '@src/modules/h5p-editor/service';
```

Comment:
Please do not use deep imports from other modules. '@src/modules/h5p-editor' also for line 8.

Reply:
solved: f818a047152c48bcbcfbf96fb384f7da1d903c10